### DISCUSSION TITLE
Outlook error 412 when I use "from email" different from default

### DISCUSSION THREAD

**Discussion by brunobarretta** [emailjs-com/help#72] [Q&A]
Hi.

I'm using emailJs with my outlook account, but when I send a e-mail with "from email" different from my default email, this error happens:

![image](https://github.com/emailjs-com/help/assets/31595067/2fee265f-eb15-4e0e-808c-61e7673f712b)

Has anyone had the same problem?

**Comment by SarKLeRoi** [comment]
me too I'm getting the same using GoDaddy Domain(outlook), when I used mine it worked (gmail)

**Comment by xr0master** [comment]
[duplicate #8](https://github.com/orgs/emailjs-com/discussions/8)
